server/neptune/temporalworker/server.go

```diff
@@ -307,6 +325,27 @@ func (s Server) buildTerraformWorker() worker.Worker {
 	return terraformWorker
 }
 
+// rebase worker only handles activites for the rebase flow
+func (s Server) buildPRRevisionWorker() worker.Worker {
+	// pass the underlying client otherwise this will panic()
+	worker := worker.New(s.TemporalClient.Client, workflows.PRRevisionTaskQueue, worker.Options{
+		WorkerStopTimeout: TemporalWorkerTimeout,
+		Interceptors: []interceptor.WorkerInterceptor{
+			temporal.NewWorkerInterceptor(),
```

Comment:
Should we be using the same interceptor? I haven't looked but i want to make sure this provides value.

Reply:
Yes, it adds latency and success/failure metrics to the `ExecuteActivity` and `ExecuteWorkflow` calls in addition to appending the workflow name to the metrics.